### DISCUSSION TITLE
When will the extension be published with the manifest in version 3?

### DISCUSSION THREAD

**Discussion by NautillusSs** [darkreader/darkreader#9193] [Q&A]
Hello, I am trying to download the CRX file for the DarkReader extension with the manifest in version 3.

Using the answers in the following question I can only obtain the manifest in version 2: [How to download a CRX file from the Chrome web store for a given ID?](https://stackoverflow.com/questions/7184793/how-to-download-a-crx-file-from-the-chrome-web-store-for-a-given-id)

```
❯ head darkreader/manifest.json
{
"update_url": "https://clients2.google.com/service/update2/crx",

    "manifest_version": 2,
    "name": "Dark Reader",
```

The URL I am using: https://clients2.google.com/service/update2/crx?response=redirect&prodversion=103.0.5060.53&acceptformat=crx2,crx3&x=id%3Deimadpbcbfnmbkopoojfekhnkhdbieeh%26uc

Is the extension currently being published as version 3? If not, would you have a timeline?

Thank you!

**Comment by Gusted** [comment]
> Is the extension currently being published as version 3?

Nope, we're still polishing out the version to be compatible with Manifest v3. We only publish it as part of our automated releases, https://github.com/darkreader/darkreader/releases/tag/v4.9.52

> If not, would you have a timeline?

🤷🏽 Depends on how much bugs we will encounter and how Firefox will be able to handle the new Manifest v3.


Why would you currently use manifest v3 over manifest v2? There's no benefit.

**Reply by osfanbuff63** [reply]
Google has already depreciated manifest v2 and will drop support in 2023.

**Reply by NautillusSs** [reply]
> Why would you currently use manifest v3 over manifest v2? There's no benefit.

I imported the extension onto my company's internal application store (took me a few months). Given the drop of support for v2, the company is requiring extensions to update to v3 internally before end of year.
If I cannot provide v3 by the end of year, they might drop the extension from the internal app store altogether.

**Reply by Gusted** [reply]
> Google has already depreciated manifest v2 and will drop support in 2023.

Still plenty of time left to complain 😝. 



> If I cannot provide v3 by the end of year, they might drop the extension from the internal app store altogether.

Like I said, we currently generate MV3 binaries for each release, https://github.com/darkreader/darkreader/releases/. Should be a no-brainer to use that.
